scripts/dockers/cb_master/configure-portal.py

```diff
@@ -256,6 +256,21 @@ def configure_user_groups(self, portalhrd):
         if not self.lcl.networkids.exists(gid):
             networkids = {"id": gid, "networkids": list(newrange)}
             self.lcl.networkids.set(networkids)
+        # configure Disk-Types
+        disktypes = [
+            ('B', 'Boot Disk', True),
+            ('M', 'Meta Data Disk', True),
+            ('D', 'Data Disk', True),
+            ('C', 'CD-ROM Disk', False),
+            ('P', 'Physical Disk', False)
+        ]
+        if self.ccl.disktype.conunt({}) == 0:
```

Comment:
typo in count.
Please add the if inside the for loop and search for a specific type so it will be easier to extend this list
Are the prefixes missing here?